specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/stable/2021-11-12/examples/HanaInstances_Get.json

```diff
@@ -0,0 +1,66 @@
+{
+  "parameters": {
+    "api-version": "2021-11-12",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "hanaInstanceName": "myHanaInstance"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.HanaOnAzure/hanaInstances/myHanaInstance",
+        "name": "myHanaInstance",
+        "type": "Microsoft.HanaOnAzure/hanaInstances",
+        "location": "westus",
+        "tags": {
+          "key": "value"
+        },
+        "properties": {
```

Comment:
We expect systemData field to be also present in the response, are we not getting it from the api response?